### PR TITLE
Lim edits: Beginning - End of Sec 2.2

### PR DESCRIPTION
**Pull request by pllim**
I ran out of time today. If I don't get a chance to comment more (which will be separate PR if I do) before Adrian sends it out, I will continue as part of the "general public" effort.

### CODE REVIEW

File: src/ms.tex [hunk 1]
```diff
@@ -203,14 +203,14 @@ \subsection{Highlighted Feature Development} \label{sec:core-features}
...
 specialized packages within the Astropy ecosystem. Even many non-astronomical
 \python libraries have come to rely on \astropypkg. The relationship between
-\astropy and the \python-using community is reciprocal, with the evolving needs
+\astropy and the broader \python community is reciprocal, with the evolving needs
 of the community and the maturation of the broader scientific \python ecosystem
 driving much of the development of the \astropypkg core package.
 
-Within \astropypkg, development may be roughly split into a few
-categories: new features added, intra- and inter-package interoperability,
-improvements to precision, accuracy, and reproducibility, and performance
```
Comment (pllim) [inline src/ms.tex:212]: Is there a difference between "precision" and "accuracy"? Do we need both?

**Comment by pllim** [comment]
p.s. My LaTeX is rusty. If I miss a `\` here and there, please let me know.

File: src/ms.tex (hunk 1 above)
Comment (adrn) [reply src/ms.tex:212]: I think below, "precision" is used to refer to numerical precision, whereas "accuracy" refers to the capacity of algorithms to produce correct values. So I think they are subtly different, but maybe we should change precision -> numerical precision?

**Comment by adrn** [comment]
Looks like this needs a rebase now 😬

**Review by nmearl — APPROVED** [review]

**Review by adrn — COMMENTED** [review]
These edits look good to me, except for the comment I left w.r.t. precision/accuracyI

File: src/ms.tex (hunk 1 above)
Comment (pllim) [reply src/ms.tex:212]: OK, added it back.

**Comment by pllim** [comment]
Rebased and added back "accuracy".

**Review by adrn — APPROVED** [review]
Looks good -- thanks @pllim!

**Comment by pllim** [comment]
Thanks for the reviews! I will submit more when I can but it should not hold off you sending out a draft to the general public.

**Pull request merged**